go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/protobuf-specs
 
-go 1.18
+go 1.21
```

Comment:
why not 1.22?

Reply:
I'd be fine even with 1.23 😄 - changed to 1.22 now